src/activity-log/index.html

```diff
@@ -52,9 +52,9 @@
                     </td>
                     <td class="u-w65pct">
                         <a href="{{ item.permalink }}">
-                            <h4 class="u-mb0">
+                            <span class="h4 u-mb0">
```

Comment:
Yeah. I tried it by putting the `th.u-visually-hidden` on the row and I got a weird top border. I'm hoping the cf-tables team can look into it to fix across all our tables.